app/app.go

```diff
@@ -1122,17 +1169,17 @@ func (app *App) GetTxConfig() client.TxConfig {
 func (app *App) Name() string { return app.BaseApp.Name() }
 
```

Comment:
Pretty sure we need the PreBlocker. "When using (legacy) application wiring, the following must be added to app.go"

```
func (app *App) PreBlocker(ctx sdk.Context, req *abci.RequestFinalizeBlock) (*sdk.ResponsePreBlock, error) {
  return app.mm.PreBlock(ctx, req)
}
```

Reply:
I've got it in the list of TODOs in the [epic](https://github.com/provenance-io/provenance/issues/1760).